tests/test_api/test_endpoints.py

```diff
@@ -173,6 +174,71 @@ def get_token():
         )
 
 
+def test_update_user():
+    mdb = get_mongo(run_config_frozen__normal_env).db
+    rs = ensure_test_resources(mdb)
+    base_url = os.getenv("API_HOST")
+
+    @retry(wait=wait_random_exponential(multiplier=1, max=60))
+    def get_token():
+        """
+
+        Randomly wait up to 2^x * 1 seconds between each retry until the range reaches 60
+        seconds, then randomly up to 60 seconds afterwards
+
+        """
+
+        _rv = requests.post(
+            base_url + "/token",
+            data={
+                "grant_type": "password",
+                "username": rs["user"]["username"],
+                "password": rs["user"]["password"],
+            },
+        )
+        token_response = _rv.json()
+        return token_response["access_token"]
+
+    headers = {"Authorization": f"Bearer {get_token()}"}
+
+    user_in1 = UserIn(username="foo", password="oldpass")
+    mdb.users.delete_one({"username": user_in1.username})
+    mdb.users.update_one(
+        {"username": rs["user"]["username"]},
+        {"$addToSet": {"site_admin": "nmdc-runtime-useradmin"}},
```

Comment:
I _think_ this test has an admin user updating its _own_ password. While that _is_ a valid use case, I am more interested in confirming that an admin user can update a _different_ user's password. I may be mistaken about what is happening in this test.